server/storage.go

```diff
@@ -122,12 +122,6 @@ func (s *BadgerStorage) Start() error {
 				return errors.Wrap(err, "failed to perform garbage collection on Badger DB")
 			}
 
-			s.logger.
-				WithFields(logrus.Fields{
-					"interval":     s.gcInterval,
-					"discardRatio": s.gcDiscardRatio,
-				}).
-				Debug("Performed garbage collection on Badger value log")
```

Comment:
this one disappeared altogether, is that ok?

Reply:
@janezpodhostnik badger is retiring soon ( #301 ), that's why I skipped this one.